src/opm/output/eclipse/FileService/EclOutput.cpp

```diff
@@ -56,6 +56,20 @@ void EclOutput::write<std::string>(const std::string& name,
     }
 }
 
+template <>
+void EclOutput::write<Opm::RestartIO::Helpers::CharArrayNullTerm<8>>
+    (const std::string&                                                name,
+     const std::vector<Opm::RestartIO::Helpers::CharArrayNullTerm<8>>& data)
```

Comment:
When the `Opm::RestartIO::Helpers::CharArrayNullTerm<8>` type is consumed publically it must get a more ergonomic name; the current name is orders of magnitude too long.